flink-connector-bigquery/src/test/java/com/google/cloud/flink/bigquery/sink/serializer/TestBigQuerySerializer.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (C) 2024 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.flink.bigquery.sink.serializer;
+
+import com.google.cloud.flink.bigquery.sink.exceptions.BigQuerySerializationException;
+import com.google.protobuf.ByteString;
+
+/** Mock serializer for Sink unit tests. */
```

Comment:
Let's call this a "Fake" to be consistent

Reply:
Done. Good catch, thanks!

Reply:
you can resolve this convo; thanks!